### PR TITLE
Clean up copier params

### PR DESCRIPTION
**Pull request by ruestefa**
- Improve some parameter descriptions
- Add new parameters `project_keywords`, `project_doc_url` and `project_ssh_url`
- Change `project_github` from partial to full github URL (enables support for other hosts)
- Add some better parameter names as comments in case renaming (as originally proposed in #83) ever happens

### CODE REVIEW

File: copier.yaml [hunk 1]
```diff
@@ -1,46 +1,69 @@
...
 
+project_keywords:
+  help: "Keywords that describe your project (comma-separated)"
+  default: "{{ project_name }}"
+  type: str
+
+# project_git_user:
+github_username:
+  help: "Name of github/gitlab user hosting the project"
+  # default: "MeteoSwiss-APN"
```
Comment (regDaniel) [inline copier.yaml:46]: remove

File: copier.yaml [hunk 2]
```diff
@@ -1,46 +1,69 @@
...
+  default: "{{ full_name.lower().replace(' ', '_') }}"
+  type: str
+
+# project_git_url:
 project_github:
-  help: "Project path on github"
-  default: "MeteoSwiss-APN/{{ project_slug }}"
+  help: "URL of project git repository"
```
Comment (regDaniel) [inline copier.yaml:52]: Default should stay MeteoSwiss-APN

**Review by regDaniel — CHANGES_REQUESTED** [review]
The default for GitHub should stay at MeteoSwiss-APN.

File: copier.yaml (hunk 1 above)
Comment (ruestefa) [reply copier.yaml:46]: Remove what exactly?

File: copier.yaml (hunk 2 above)
Comment (ruestefa) [reply copier.yaml:52]: It is if the default `github_username` is set back to 'MeteoSwiss-APN'.

File: copier.yaml (hunk 1 above)
Comment (regDaniel) [reply copier.yaml:46]: I was originally referring to the old default, which was commented out. But Given the other answer, I'll set the default here back to MeteoSwiss-APN and delete the other default.

File: copier.yaml (hunk 2 above)
Comment (regDaniel) [reply copier.yaml:52]: see comment above.

**Comment by regDaniel** [comment]
launch jenkins

**Pull request merged**